ccgains/trades.py

```diff
@@ -257,9 +267,43 @@ def _parse_trade(str_list, param_locs, default_timezone):
         else:
             pdict[key] = val
 
+    # Get missing fees if requested
+    if fill_missing_fees and pdict['kind'] == 'Withdrawal':
```

Comment:
Just want to mention here something about `pdict['kind'] == 'Withdrawal'`:
This will only work because the only two wallets so far that actually use `fill_missing_fees` (Trezor and Electrum) will during import actually call withdrawals `'Withdrawal'`. But some exchanges call withdrawals differently or use a different language or whatever, so this might become an issue if one wants to use this function at some point with a different wallet. That is the reason why I always checked if `buy_amount` is zero to make sure we are dealing with a withdrawal. (Latest developments introduced 'Payment' kind, so this also is not the best way anymore). Anyway, we can keep this line just as it is, but would prefer a cleaner solution. Any suggestions how to improve the issue? Maybe enforcing calling withdrawals `'Withdrawal'` during import for all wallets/exchanges?

Reply:
*Sorry, Electrum only. But my question still stands. :-)